ballerina/Ballerina.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 org = "ballerina"
 name = "observe"
-version = "1.0.3"
+version = "1.0.4"
 distribution = "slbeta3"
 export = ["observe", "observe.mockextension" ]
 
 [[platform.java11.dependency]]
-path = "../native/build/libs/observe-native-1.0.3.jar"
+path = "../native/build/libs/observe-native-1.0.4-SNAPSHOT.jar"
```

Comment:
Is this correct ?

Reply:
Yeah this is an [automated commit](https://github.com/ballerina-platform/module-ballerina-observe/pull/37/commits/401e8ab0647ea90892f37bd652eed5a6faefc503) done through gradle.

It replaces the https://github.com/ballerina-platform/module-ballerina-observe/blob/main/build-config/resources/Ballerina.toml file with the versions in gradle.properties file automatically and commits them.